Makefile

```diff
@@ -56,14 +56,14 @@ install:
 	GO111MODULE=on go install \
 		-trimpath \
 		-ldflags \
-		"-X github.com/onflow/flow-cli/build.commit=$(COMMIT) -X github.com/onflow/flow-cli/build.semver=$(VERSION) -X github.com/onflow/flow-cli/pkg/flowkit/util.MIXPANEL_SERVICE_ACCOUNT_SECRET=$(MIXPANEL_SERVICE_ACCOUNT_SECRET) -X github.com/onflow/flow-cli/pkg/flowkit/util.MIXPANEL_PROJECT_TOKEN=$(MIXPANEL_PROJECT_TOKEN)" \
+		"-X github.com/onflow/flow-cli/build.commit=$(COMMIT) -X github.com/onflow/flow-cli/build.semver=$(VERSION) -X github.com/onflow/flow-cli/pkg/flowkit/util.MIXPANEL_SERVICE_ACCOUNT_SECRET=${MIXPANEL_SERVICE_ACCOUNT_SECRET} -X github.com/onflow/flow-cli/pkg/flowkit/util.MIXPANEL_PROJECT_TOKEN=${MIXPANEL_PROJECT_TOKEN}" \
```

Comment:
This is not the problem. The problem is that these variables weren't set as part of the environment running this make command. Please set them https://github.com/onflow/flow-cli/blob/3284a5b28330350bffbcdea1a2b4f6b84cf05c3e/.github/workflows/cd.yml#L30 and please test this as part of the PR.

Reply:
I think we can run the test as part of CI, so we set the env variables with the secrets for make test, to make sure that they are there.